src/cider/nrepl/middleware/test.clj

```diff
@@ -72,7 +76,7 @@
         c (when (seq test/*testing-contexts*) (test/testing-contexts-str))
         i (count (get-in (@current-report :results) [ns (:name (meta v))]))
         gen-input (:gen-input @current-report)
-        pprint-str #(with-out-str (pp/pprint %))]
+        pprint-str #(with-out-str (if (custom-print-method? %) (pp/pprint %) (println %)))]
```

Comment:
Might also be a good idea to add some comment explaining the need for this, and ideally a unit test.

Reply:
@bbatsov done! can you check again?